x-pack/plugins/alerting/common/rule_tags_aggregation.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  AggregationsAggregationContainer,
+  AggregationsCompositeAggregation,
+  AggregationsAggregateOrder,
+} from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import type { AggregateOptions } from './rule';
+
+export type RuleTagsAggregationOptions = Pick<
```

Comment:
Like we discussed, this type is not used!

Reply:
This is used in the rule tags route now